docs/040-install_intro.md

```diff
@@ -20,10 +20,25 @@ The easiest and fastest way to experience what OVOS has to offer is to use one o
 
 **NOTE** Images are currently only available for a RPi3b/b+/4.  More may be on the way.
 
-- [Buildroot-ovos](https://drive.google.com/drive/folders/113-zmx6ncoeLNsayseNxoaTlaAk1AfU2)
-  - The most complete and advanced image OVOS provides, complete with a default set of skills and a GUI.
-- [raspbian-ovos](https://ovosimages.ziggyai.online/raspbian/development)
-  - The newest image from the OVOS team.  This is a "headless" image (NO GUI), but comes with a preinstalled set of skills also.  This image will provide you with an experience similar to the origional [picroft](https://github.com/MycroftAI/enclosure-picroft)
+#### Features across all images
+
+- Out of the box support for most USB microphones and speakers.
+- Out of the box support for ReSpeaker style HAT's.
+- Out of the box support for the Mycroft SJ201 sound card.
+- More hardware support to come.
+- A small set of pre-installed skills so you may use your assistant right away
```

Comment:
I suggest reordering these, so that readers see the thing about skills before they see something that might not apply to them (and start skimming)

- Skills
- OOTB MkII support
- More to come